webscrape.py

```diff
@@ -0,0 +1,23 @@
+from bs4 import BeautifulSoup
+import requests
+
+url = 'http://www.faa.gov/air_traffic/flight_info/aeronav/digital_products/vfr/?viewType=Print&viewClass=Print'
+response = requests.get(url)
+html = response.text
+soup = BeautifulSoup(html, 'lxml')
+table = soup.find(class_='striped')
+
+map_data = []
+temp = open("temp", 'w+')
+temp.truncate()
+
+for row in table.find_all('tr')[1:]:
+    col = row.find_all('td')
+
+    city = col[0].string.strip()
```

Comment:
Might want to add some error handling to check that were getting the right information and also you could use that to let us know when/if the html of the webpage changes since that could potential give us some bugs in the future.